src/core-sdk-tasks/core-sdk-tasks.csproj

```diff
@@ -18,6 +18,8 @@
     <PackageReference Include="System.Reflection.Metadata" Version="8.0.0" />
     <PackageReference Include="WindowsAzure.Storage" Version="9.3.3" Condition="'$(DotNetBuildFromSource)' != 'true'" />
     <PackageReference Include="System.Net.Http" Version="4.3.4" />
+    <PackageReference Include="System.Formats.Asn1" Version="$(SystemFormatsAsn1PackageVersion)"/>
```

Comment:
@marcpopMSFT,
I saw you removed two packagerefs to this package in the SDK repo and said they weren't necessary in 8.0 because of CPM. Should we remove this reference as well?

Reply:
I confirmed that CPM is not being used. Therefore, we might need to manage package references manually.

Reply:
Jason is correct. We never onboarded to CPM in the installer repo (with transitive pinning) so we need the package references here. That's one of the reasons CPM is valuable.